packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/Events/index.js

```diff
@@ -0,0 +1,288 @@
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+import { FieldLabel, Flex, Typography, BaseCheckbox, Checkbox } from '@strapi/design-system';
+
+import { useModels } from '../../../../../../../hooks';
+
+export const formatValue = (value) =>
+  value.reduce((acc, curr) => {
+    const key = curr.split('.')[0];
+
+    if (!acc[key]) {
+      acc[key] = [];
+    }
+    acc[key].push(curr);
+
+    return acc;
+  }, {});
+
+const StyledTable = styled.table`
+  td {
+    height: ${52 / 16}rem;
+    width: 10%;
+    vertical-align: middle;
+    text-align: center;
+  }
+
+  tbody tr:nth-child(odd) {
+    background: ${({ theme }) => theme.colors.neutral100};
+  }
+
+  tbody tr td:first-child {
+    padding-left: ${({ theme }) => theme.spaces[7]};
+  }
+`;
+
+const getCEHeaders = (isDraftAndPublish) => {
+  if (isDraftAndPublish) {
+    return [
+      { id: 'Settings.webhooks.events.create', defaultMessage: 'Create' },
+      { id: 'Settings.webhooks.events.update', defaultMessage: 'Update' },
+      { id: 'app.utils.delete', defaultMessage: 'Delete' },
+      { id: 'app.utils.publish', defaultMessage: 'Publish' },
+      { id: 'app.utils.unpublish', defaultMessage: 'Unpublish' },
+    ];
+  }
+
+  return [
+    { id: 'Settings.webhooks.events.create', defaultMessage: 'Create' },
+    { id: 'Settings.webhooks.events.update', defaultMessage: 'Update' },
+    { id: 'app.utils.delete', defaultMessage: 'Delete' },
+  ];
+};
+
+const getCEEvents = (isDraftAndPublish) => {
+  if (isDraftAndPublish) {
+    return {
+      entry: ['entry.create', 'entry.update', 'entry.delete', 'entry.publish', 'entry.unpublish'],
+      media: ['media.create', 'media.update', 'media.delete'],
+    };
+  }
+
+  return {
+    entry: ['entry.create', 'entry.update', 'entry.delete'],
+    media: ['media.create', 'media.update', 'media.delete'],
+  };
+};
+
+const Root = ({ children }) => {
+  const { formatMessage } = useIntl();
+  const { collectionTypes } = useModels();
+
+  const isDraftAndPublish = React.useMemo(
+    () => collectionTypes.some((ct) => ct.options.draftAndPublish === true),
+    [collectionTypes]
+  );
+
+  const childrenWithProps = React.Children.map(children, (child) => {
+    if (React.isValidElement(child)) {
+      return React.cloneElement(child, { isDraftAndPublish });
+    }
+
+    return child;
+  });
+
+  return (
+    <Flex direction="column" alignItems="stretch" gap={1}>
+      <FieldLabel>
+        {formatMessage({
+          id: 'Settings.webhooks.form.events',
+          defaultMessage: 'Events',
+        })}
+      </FieldLabel>
+      <StyledTable>{childrenWithProps}</StyledTable>
+    </Flex>
+  );
+};
+
+Root.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const Headers = ({ getHeaders = getCEHeaders, isDraftAndPublish = false }) => {
+  const { formatMessage } = useIntl();
+  const headers = getHeaders(isDraftAndPublish);
+
+  return (
+    <thead>
+      <tr>
+        <td />
```

Comment:
Same as @gu-stav lets use the `Table` component it has the accessibility stuff built in e.g. keyboard navigation 👍🏼